src/main/scala/fi/oph/koski/omadataoauth2/OmaDataOAuth2Repository.scala

```diff
@@ -99,7 +100,8 @@ class OmaDataOAuth2Repository(val db: DB) extends DatabaseExecutionContext with
               AccessTokenSuccessResponse(
                 accessTokenAttempt,
                 "Bearer",
-                LocalDateTime.now.until(row.voimassaAsti.toLocalDateTime, ChronoUnit.SECONDS).max(0)
+                LocalDateTime.now.until(row.voimassaAsti.toLocalDateTime, ChronoUnit.SECONDS).max(0),
+                row.voimassaAsti.toLocalDateTime.format(DateTimeFormatter.ISO_DATE_TIME)
```

Comment:
Tämä generoi nyt aikaleiman ilman aikavyöhykettä, joka olettaa paikallisen (Suomen) ajan. Vois ehkä laittaa se timezone tieto sinne kans, mutta katoin että muuallakin Koskessa toimitaan näin